pyanaconda/modules/network/initialization.py

```diff
@@ -145,18 +152,16 @@ def _find_initramfs_connection_of_iface(self, iface):
 class DumpMissingConfigFilesTask(Task):
     """Task for dumping of missing config files."""
 
-    def __init__(self, nm_client, default_network_data, ifname_option_values):
+    def __init__(self, default_network_data, ifname_option_values):
         """Create a new task.
 
-        :param nm_client: NetworkManager client used as configuration backend
-        :type nm_client: NM.Client
         :param default_network_data: kickstart network data of default device configuration
         :type default_network_data: NetworkData
         :param ifname_option_values: list of ifname boot option values
         :type ifname_option_values: list(str)
         """
         super().__init__()
-        self._nm_client = nm_client
+        self._nm_client = None
```

Comment:
@rvykydal I think we should eventually drop `self._nm_client` and start to propagate a local variable instead. However, that's for an follow-up pull request.

Reply:
Addressed in https://github.com/rhinstaller/anaconda/pull/4322
https://github.com/rhinstaller/anaconda/pull/4322/commits/a8c10e93e5308d6797e834f2d541beffe6c08eef